README.md

```diff
@@ -4,7 +4,7 @@
 
 # Twilio Video Quickstart for Objective-C
 
-> NOTE: These sample applications use the Twilio Video 1.x APIs. For examples using our 2.0.0-preview APIs, please see the [2.0.0-preview](https://github.com/twilio/video-quickstart-objc/tree/2.0.0-preview) branch.
+> NOTE: These sample applications use the Twilio Video 1.x APIs. For examples using our 2.0.0-beta1 APIs, please see the [master](https://github.com/twilio/video-quickstart-objc/tree/master) branch.
```

Comment:
Lets use 2.0.0-beta so that we don't need to update this branch every time we make a release.